src/main/resources/application.properties

```diff
@@ -5,6 +5,4 @@ spring.mail.properties.mail.smtp.connectiontimeout = 5000
 spring.mail.properties.mail.smtp.timeout = 5000
 spring.mail.properties.mail.smtp.writetimeout = 5000
 spring.mail.properties.mail.smtp.starttls.enable=true
-spring.mail.host = smtp.gmail.com
-spring.mail.port = 587
-spring.mail.username = covidtracking08@gmail.com
\ No newline at end of file
+spring.mail.port = 587
```

Comment:
you may want to find a way to remove the mentions of the email address from all history. Maybe debase can do this?

Reply:
I will delete the branch and double check. If this record is still visible, I will rebase.